CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Fixed
```

Comment:
## [1.4.2] - 2020-02-13
### Added
- New method `cartItemQuantitySet()` to update quantity of a cart item directly.